src/type/preload.ts

```diff
@@ -640,3 +644,12 @@ export interface MessageBoxReturnValue {
 }
 
 export const SandboxKey = "electron" as const;
+
+/** プロジェクト一時ファイル */
+export type TempProjectType = {
```

Comment:
将来バージョンアップなどでこのデータ構造は変わった時でも正常にファイルが読める必要があり、マイグレーション処理が必要なのですが、なんか忘れそうなのでコメントでメモだけお願いしても良いでしょうか 🙇 
```suggestion
/**
 * プロジェクト一時ファイル
 * 変更時はマイグレーションが必要
 */
export type TempProjectType = {
```